docs/sources/db.md

```diff
@@ -0,0 +1,17 @@
+# Triage database provisioner
+
+This document explains the purpose and behavior of the Triage database provisioner, accessed from the Triage CLI. It is optional and only intended for use if you don't have an existing Postgres database to use for Triage.
+
+The Triage database provisioner is just a single command:
+
+`triage db up`
+
+This command attempts to use docker to spawn a new Postgres 12 database. If successful, it will prompt you for a password to use for a user, and populate the connection information in database.yaml. The next time you run `triage db up`, it will look for the existing container and reuse it.
+
```

Comment:
Might be useful to provide some info/link out to installing `psql` and/or a GUI like `dbeaver` for people who aren't too familiar with using postgres generally?